klarna-on-demand/src/main/java/com/klarna/ondemand/Context.java

```diff
@@ -24,4 +26,29 @@ protected static String getApiKey(){
         }
         return apiKey;
     }
+
+    /**
+     * Sets the button color for Klarna views in this application.
+     * @param buttonColor Merchant's public API key for this application.
+     */
+    public static void setButtonColor(Integer buttonColor) { Context.buttonColor = buttonColor; }
```

Comment:
Is it customary for colors to be an integer? How does that work, exactly? Some sort of RGB thing?